src/Edm/Validation/Internal/InterfaceValidator/VisitorOfIStringValue.php

```diff
@@ -10,10 +10,10 @@
 
 class VisitorOfIStringValue extends VisitorOfT
 {
-    protected function VisitT($value, array &$followup, array &$references): iterable
+    protected function VisitT($value, array &$followup, array &$references): ?iterable
     {
         assert($value instanceof IStringValue);
-        return $value->getValue() == null ?
+        return null === $value->getValue() ?
```

Comment:
Yet more nullability confusion, @c-harris .  Treated as nullable here, treated as not nullable in EdmValueWriter::StringAsXml, EdmModelCsdlSchemaWriter->WriteRequiredAttribute, xmlreader->XML, ExpressionTypeChecker::TryAssertStringConstantAsType,  nullability ambiguous when passed to xmlwriter->writeRaw & xmlwriter->text.